node/src/actors/chain_manager/mod.rs

```diff
@@ -1224,6 +1225,10 @@ impl ChainManager {
                         act.chain_state.chain_info.as_mut().unwrap().highest_superblock_checkpoint =
                             act.chain_state.superblock_state.get_beacon();
 
+                        // Let JSON-RPC clients know that the blocks in the previous superblock can now
+                        // be considered consolidated
+                        act.notify_previous_superblock_consolidation(ctx);
+
```

Comment:
It is weird that there are some format issues. For example this match case is not properly indented. :)